build/Build.csproj

```diff
@@ -33,8 +33,18 @@
     <PackageReference Include="Cake.FileHelpers" Version="7.0.0" />
     <PackageReference Include="Cake.Frosting" Version="5.0.0" />
     <PackageReference Include="Cake.Git" Version="4.0.0" />
+    <PackageReference Include="Microsoft.Direct3D.DXC" Version="1.8.2505.32" />
+    <PackageReference Include="MonoGame.Tool.Dxc" Version="1.8.2505.7" />
     <PackageReference Include="NuGet.Packaging" Version="6.11.1" />
     <PackageReference Include="System.Formats.Asn1" Version="9.0.0" />
   </ItemGroup>
 
+  <Target Name="CopyNuGetToolFiles" AfterTargets="Build">
+    <ItemGroup>
+      <ToolFiles Include="$(NuGetPackageRoot)Microsoft.Direct3D.DXC\1.8.2505.32\build\native\bin\x64\*.dll" />
+	    <ToolFiles Include="$(NuGetPackageRoot)Microsoft.Direct3D.DXC\1.8.2505.32\build\native\bin\x64\*.exe" />
```

Comment:
I don't like the fact that we have version numbers in the path there, it's an opportunity to forget to update this when updating the nuget...

... but I assume this will not be a problem anymore since we're not relying on those nugets directly in the future, right?